src/shared/lib/utils/responseTypes/index.ts

```diff
@@ -0,0 +1,21 @@
+import { ResponseType } from '@app/shared/api/services/ActivityItemDto';
+
+import {
+  appSupportedResponseTypes,
+  universalSupportedResponseTypes,
+  webSupportedResponseTypes,
+} from '../../constants/responseTypes';
+
+export const getIsAppOnly = (responseType: ResponseType) =>
```

Comment:
Nit: I know this repo is called `mindlogger-app-refactor`, but I always thought that was a misnomer because all three FE projects are indeed apps – this one's a mobile app, while the other two are web apps. Would you be all right with using the language "mobile" here instead? I think we're already using that language in the Admin App.

```suggestion
export const getIsMobileOnly = (responseType: ResponseType) =>
```

Reply:
Updated [here](https://github.com/ChildMindInstitute/mindlogger-app-refactor/pull/873/commits/acb8c088b64e02c11ece0aea244753383474a503)!